src/canvas.class.js

```diff
@@ -892,7 +892,7 @@
      */
     searchPossibleTargets: function (objects, pointer) {
       var target = this._searchPossibleTargets(objects, pointer);
-      return target;
+      return target && target.interactive && this.targets[0] ? this.targets[0] : target;
```

Comment:
note for myself: check what happens with nested groups

Reply:
If I understand it correctly it iterates over all objects from last to first, wee it hits an objects it walks down it's objects as well so I expect targets[0] to be the top most target to accept the pointer.